bindings/osrm_c89.c

```diff
@@ -84,7 +84,7 @@ int main(int argc, char** argv) {
 
   /* If we got here on a failure path notify the user */
   if (error) {
-    fprintf(stderr, "Error: %s\n", osrmc_error_message(error));
+    fprintf(stderr, "Error: code=%s, message=%s\n", osrmc_error_code(error), osrmc_error_message(error));
```

Comment:
Seeing this change in the example bindings here: have you thought about ABI/API testing `libosrm -> libosrmc -> your rust bindings`? Right now this library doesn't really come with e.g. CI/CD or integration tests. Working on the C bindings might break the examples and/or your rust bindings. Just some food for thought.

Reply:
There are tests for the Rust bindings, but they're unfortunately not running in publicly visible CI right now. 

I consider the `libosrmc` bindings an implementation detail of the Rust bindings. A specific version is included as a git submodule, and it's then statically compiled into the Rust library. Thankfully we don't need to worry about working with arbitrary `libosrmc`s!

The `libosrm` is a bit trickier: we test against the specific version that we need, but we don't test all available versions. I figured I'd cross that bridge if anyone else came to depend on the Rust bindings. 🙂